eldritch/test_mythos.py

```diff
@@ -1470,6 +1471,85 @@ def setUp(self):
       monster.idx = idx
       monster.place = self.state.monster_cup
 
+  def testActivateRumor13(self):
+    self.state.mythos.append(Mythos13())
+    self.state.event_stack.append(Mythos(self.char))
+    self.resolve_until_done()
+    self.assertTrue(self.state.rumor)
+    self.assertEqual(self.state.rumor.name, "Mythos13")
+    self.assertFalse(self.state.rumor.failed)
+    self.assertNotIn(self.state.rumor, self.state.mythos)
+
+  def testRumor13Progress(self):
+    self.state.event_stack.append(IncreaseTerror(6))
+    self.resolve_until_done()
+    self.state.mythos.append(Mythos13())
+    self.state.event_stack.append(Mythos(self.char))
+    self.resolve_until_done()
+    self.assertEqual(self.state.rumor.name, "Mythos13")
+    self.assertEqual(self.state.terror, 6)
+
+    self.advance_turn(self.state.turn_number+1, "mythos")
+    self.assertTrue(self.state.event_stack)
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(side_effect=[3])):
+      self.resolve_until_done()
+    self.assertEqual(self.state.rumor.name, "Mythos13")
+    self.assertEqual(self.state.terror, 6)
+    self.assertFalse(self.state.rumor.failed)
+
+    self.advance_turn(self.state.turn_number+1, "mythos")
+    self.assertTrue(self.state.event_stack)
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(side_effect=[1])):
+      self.resolve_until_done()
+    self.assertEqual(self.state.rumor.name, "Mythos13")
+    self.assertEqual(self.state.terror, 7)
+    self.assertFalse(self.state.rumor.failed)
+
+    self.advance_turn(self.state.turn_number+1, "mythos")
+    self.assertTrue(self.state.event_stack)
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(side_effect=[3])):
+      self.resolve_until_done()
+    self.assertEqual(self.state.rumor.name, "Mythos13")
+    self.assertEqual(self.state.terror, 7)
+    self.assertFalse(self.state.rumor.failed)
+
+  def testFailRumor13(self):
+    rumor = Mythos13()
+    self.state.mythos.append(rumor)
+    self.state.event_stack.append(Mythos(self.char))
+    self.resolve_until_done()
+    self.assertEqual(self.state.rumor.name, "Mythos13")
+    self.state.event_stack.append(IncreaseTerror(9))
+    self.resolve_until_done()
+
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(side_effect=[1, 2])):
```

Comment:
If it only takes one more increase of the terror level for the rumor to fail, why are there two values for the dice roll here?

Reply:
I see you spotted a place where I copy-pasted from elsewhere poorly.